packages/portal-proto/src/components/DownloadButtons/DownloadButton.tsx

```diff
@@ -36,7 +36,7 @@ interface DownloadButtonProps {
   inactiveText: string;
   activeText: string;
   filename?: string;
-  size?: number;
+  downloadSize?: number;
```

Comment:
I had to change the name of this prop because it was clashing with the size prop on the [mantine button component](https://mantine.dev/core/button/?t=props). We weren't actually using this prop anywhere though so easy update :/